AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/MySql/Converters/PermissionConverter.cs

```diff
@@ -2,32 +2,14 @@
 
 namespace AppSettingsManagerApi.Infrastructure.MySql.Converters;
 
-public class PermissionBiverter : IBidirectionalConverter<Model.Permission, Permission>
+public class PermissionConverter : IBidirectionalConverter<Model.Permission, Permission>
 {
-    private readonly IBidirectionalConverter<Model.User, User> _userBiverter;
-
-    private readonly IBidirectionalConverter<
-        Model.SettingGroup,
-        SettingGroup
-    > _settingGroupBiverter;
-
-    public PermissionBiverter(
-        IBidirectionalConverter<Model.User, User> userBiverter,
-        IBidirectionalConverter<Model.SettingGroup, SettingGroup> settingGroupBiverter
-    )
-    {
-        _userBiverter = userBiverter;
-        _settingGroupBiverter = settingGroupBiverter;
-    }
-
```

Comment:
Ah, this would have created circular references right? If the SettingGroupConverter referenced this converter and this converter also referenced SettingGroupConverter